apps/web/src/server/views/auth/unauthorized.njk

```diff
@@ -0,0 +1,9 @@
+{% extends "app/layouts/app-two-column.layout.njk" %}
+
```

Comment:
You need to define the pageTitle, else the browser tab will just have something generic unrelated to the page.
`{% set pageTitle = 'Sorry, there is a problem with your login' %}`

By setting a page title, the page layout will automatically introduce that same page title into an <h1> element, which as per your template below, will result in double headings. You can either drop the h1 in your `pageContent`, or override the inserted heading using the `pageHeading` block.